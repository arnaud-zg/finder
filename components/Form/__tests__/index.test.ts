import * as FormIndex from '../'

describe('Form', () => {
  it('should take a snapshot of FormIndex', () => {
    expect(FormIndex).toMatchSnapshot()
  })
})
