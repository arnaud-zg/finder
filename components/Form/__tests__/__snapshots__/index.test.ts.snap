// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form should take a snapshot of FormIndex 1`] = `
Object {
  "ErrorField": [Function],
  "NumberInput": [Function],
  "Select": [Function],
  "TextArea": [Function],
  "TextInput": [Function],
}
`;
